domain/domain.go

```diff
@@ -1460,7 +1460,11 @@ func (do *Domain) InitDistTaskLoop(ctx context.Context) error {
 }
 
 func (do *Domain) distTaskFrameworkLoop(ctx context.Context, taskManager *storage.TaskManager, schedulerManager *scheduler.Manager, serverID string) {
-	schedulerManager.Start()
+	err := schedulerManager.Start()
```

Comment:
Once met error, do we will retry?

Reply:
> Once met error, do we will retry?

No, this err happens when the tikv or pd is not alive.  The whole tidb will not bootstrapped.

Reply:
if only because TiKV or PD are in HA(restart), I thing retrying several times(e.g.. 3) will be better.

Reply:
Fixed